automation/capitanodoc/markdown.ts

```diff
@@ -24,7 +41,7 @@ export function renderCommand(command: Command) {
 export function renderCategory(category: Category) {
 	let result = `# ${category.title}\n`;
 
-	for (let command of category.commands) {
+	for (const command of category.commands) {
```

Comment:
Purely a comment, but there's a load over the source where you might well be better served by utilising lodash for a one-liner:
```
_.each(category.commands, (command) => result += `\n${renderCommand(command)}`;);
```

Reply:
Fair enough. `prettier` strikes again... :)

Reply:
It's slower, double the time for 1000 strings over a million passes:
```
for-of took 12562 ms
_.each() took 21359 ms
```
I mean, nothing that'd affect this, but still. Interesting.